package.json

```diff
@@ -95,6 +95,8 @@
     "babel-eslint": "^10.0.1",
     "babel-jest": "^24.3.1",
     "babel-loader": "^8.0.5",
+    "browserslist": "^4.6.6",
```

Comment:
I'm a little unhappy to see us directly depend on these, as it's not obvious why we include them, and we can't add comments to our package.json file.

Reply:
yeah :(